seed/models/meters.py

```diff
@@ -98,6 +101,9 @@ class Meter(models.Model):
         (POTABLE_INDOOR, "Potable Indoor"),
         (POTABLE_OUTDOOR, "Potable Outdoor"),
         (POTABLE_MIXED, "Potable: Mixed Indoor/Outdoor"),
+        (HEATING_DEGREE_DAY, "Heating Degree Day"),
+        (COOLING_DEGREE_DAY, "Cooling Degree Day"),
+        (TEMPERATURE, "Temperature"),
```

Comment:
Would we want to call this "average temperature"? Or do you think that would be implied well enough? 

Also, I think it is "Heating/Cooling Degree Days" as in plural...